web/app/js/components/util/MetricUtils.js

```diff
@@ -31,10 +31,18 @@ export const getPodsByDeployment = pods => {
     .reject(p => _.isEmpty(p.deployment) || p.controlPlane)
     .groupBy('deployment')
     .map((componentPods, name) => {
+      _.remove(componentPods, p => {
+        return p.status === "Terminating" || p.status === "Completed";
+      });
+
+      let podsWithStatus = _.map(componentPods, p => {
+        return _.merge({}, p, { value: p.added ? "good" : "neutral" });
+      });
+
       return {
         name: name,
         added: _.every(componentPods, 'added'),
-        pods: componentPods
+        pods: _.sortBy(podsWithStatus, "name")
       };
     })
     .sortBy('name')
```

Comment:
Given that it's possible to filter out all pods, I think we should also remove all deployments that contain 0 pods after filtering. Otherwise they'll show up as blank rows in the table. For instance:

![image](https://user-images.githubusercontent.com/9226/36346589-92209baa-13f5-11e8-9e50-8addcc004552.png)

I'd prefer if we remove the `mysql-init` row from that output, with:

```
    .groupBy('deployment')
    .map(...)
    .reject(p => _.isEmpty(p.pods))
    .sortBy('name')
    ...
```